app/models/manageiq/providers/ansible_tower/automation_manager/credential/api_create.rb

```diff
@@ -0,0 +1,36 @@
+class ManageIQ::Providers::AnsibleTower::AutomationManager::Credential
+  module ApiCreate
+    def create_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      credential = manager.with_provider_connection do |connection|
+        connection.api.credentials.create!(params)
+      end
+
+      # Get the record in our database
+      # TODO: This needs to be targeted refresh so it doesn't take too long
+      EmsRefresh.queue_refresh(manager, nil, true) if !manager.missing_credentials? && manager.authentication_status_ok?
```

Comment:
This PR is following the pattern established in #13495.  And API create is the key goal of this PR.  The modeling code will need refresh once GregB's new models PR is merged. 

I can close this for now and re-create once the new pattern is established -- @bdunne is there a PR for the new changes?